src/client-helpers.ts

```diff
@@ -200,7 +200,8 @@ export const cleanId = (path: string, method: string, suffix: string) => {
   return [method]
     .concat(path.split("/"))
     .concat(suffix)
-    .map((entry) => entry.replace(/[^A-Z0-9]/i, ""))
+    .map((entry) => entry.split(/[^A-Z0-9]/gi))
```

Comment:
the addition of the `g` allows the clean function to affect all invalid characters
The use of `.split` followed by `.flat` allows for the `CapitalCasing` that happens next to correctly map these `special-character_separated` words into capital case correctly: `SpecialCharacterSeparated`

Reply:
ok

Reply:
there is also `flatMap`, which is described are more efficient

> It is identical to a map() followed by a flat() of depth 1 , but slightly more efficient than calling those two methods separately.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap